hazelcast/src/main/java/com/hazelcast/map/impl/operation/ClearBackupOperation.java

```diff
@@ -33,10 +33,6 @@ public ClearBackupOperation(String name) {
 
     @Override
     public void run() {
-        // clear near-cache also on this backup operation, there is a possibility that no owner partition exists on this
-        // node but a near-cache exists.
-        clearLocalNearCache();
```

Comment:
Should be tested via `NearCacheTest.testMapClear_clears_localNearCache`.

:+1: